static/app/views/insights/common/views/spans/selectors/actionSelector.tsx

```diff
@@ -15,6 +15,7 @@ import {useSpansQuery} from 'sentry/views/insights/common/queries/useSpansQuery'
 import {buildEventViewQuery} from 'sentry/views/insights/common/utils/buildEventViewQuery';
 import {QueryParameterNames} from 'sentry/views/insights/common/views/queryParameters';
 import {EmptyContainer} from 'sentry/views/insights/common/views/spans/selectors/emptyOption';
+import {SupportedDatabaseSystem} from 'sentry/views/insights/database/utils/constants';
```

Comment:
unrelated to this PR, but perhaps these constants should be moved to `database/settings.ts`?

Reply:
Maybe, but I don't see what advantages in discoverability that would bring. It makes more sense to me to leave them as constants, since these aren't being used as 'settings' in any way and they are exactly that, just constant values

Reply:
Good point, when I read `supportedDatabaseSystem`, i thought its where we defined what is supported and what isn't which felt like a settings. But it isn't really that so what you said makes sense